CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## v1.18.0
+- [core] `setTopPanelVisibily` renamed to `setTopPanelVisibility`
+
```

Comment:
@5hraddha the changes will likely need to be rebased since I added an update for a `1.17.1` patch release. Also, please look at past examples of breaking change entries on how the format should be (ex: it should be under an appropriate 'breaking changes' heading for the respective release).

Reply:
@vince-fugnitto Thanks for your patience!! Have rebased the changes and updated the breaking change entry in the correct format. Thank you!